test/e2e/flowaggregator_test.go

```diff
@@ -282,8 +287,8 @@ func checkIntraNodeFlows(t *testing.T, data *TestData, podAIPs, podBIPs *PodIPs,
 	}
 }
 
-func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs, podEIPs *PodIPs, isIPv6 bool) {
-	svcB, svcC, err := createPerftestServices(data, isIPv6)
+func testHelper(t *testing.T, data *TestData, isIPv6 bool) {
+	_, svcB, svcC, svcD, svcE, err := createPerftestServices(data, false)
```

Comment:
did you mean to hardcode the 2nd argument (`isIPv6`) to false?

Reply:
No, I was doing some test and I think I forget to change it back. Thanks.